src/libstd/path.rs

```diff
@@ -184,6 +184,11 @@ pub enum Prefix<'a> {
     /// Prefix `C:` for the given disk drive.
     #[stable(feature = "rust1", since = "1.0.0")]
     Disk(#[stable(feature = "rust1", since = "1.0.0")] u8),
+
+    /// Scheme `file:` used on Redox
+    #[cfg(target_os = "redox")]
+    #[stable(feature = "rust1", since = "1.0.0")]
+    Scheme(#[stable(feature = "rust1", since = "1.0.0")] &'a OsStr),
```

Comment:
Would it be possible to defer this to a later date? I don't think this modification is something the libs team is comfortable with just yet.

Reply:
Yes, this can be deferred